platform/osx/detect.py

```diff
@@ -142,23 +141,16 @@ def configure(env):
                 ]
             )
             env.Append(LINKFLAGS=["-fsanitize=undefined"])
-            if env["use_llvm"]:
-                env.Append(
-                    CCFLAGS=[
-                        "-fsanitize=nullability-return,nullability-arg,function,nullability-assign,implicit-integer-sign-change,implicit-signed-integer-truncation,implicit-unsigned-integer-truncation"
-                    ]
-                )
-            else:
-                env.Append(CCFLAGS=["-fsanitize=bounds-strict"])
+            env.Append(
+                CCFLAGS=[
+                    "-fsanitize=nullability-return,nullability-arg,function,nullability-assign,implicit-signed-integer-truncation,implicit-unsigned-integer-truncation"
```

Comment:
Can you remove `implicit-unsigned-integer-truncation` and `implicit-signed-integer-truncation` from Linux, Mac and Server build?
I was almost sure that I removed that flags, but looks that they still exists here.
They cause error spam just like you show at the top:
```
thirdparty/misc/pcg.cpp:12:27: runtime error: implicit conversion from type 'unsigned long long' of value 111260437319 (64-bit, unsigned) to type 'uint32_t' (aka 'unsigned int') changed the value to 3886254919 (32-bit, unsigned)
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior thirdparty/misc/pcg.cpp:12:27 in

core/variant/variant.cpp:2852:11: runtime error: implicit conversion from type 'int64_t' (aka 'long long') of value -1 (64-bit, signed) to type 'uint32_t' (aka 'unsigned int') changed the value to 4294967295 (32-bit, unsigned)
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior core/variant/variant.cpp:2852:11 in

thirdparty/nanosvg/nanosvg.h:2190:49: runtime error: division by zero
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior thirdparty/nanosvg/nanosvg.h:2190:49 in
``` 
With this flags it is very hard to find real errors in code

Reply:
> Can you remove implicit-unsigned-integer-truncation and implicit-signed-integer-truncation from Linux, Mac and Server build?

Done.